api/v2.0/swagger.yaml

```diff
@@ -1789,6 +1842,40 @@ paths:
           $ref: '#/responses/401'
         '500':
           $ref: '#/responses/500'
+  /projects/{project_name}/log-exts:
+    get:
+      summary: Get recent logs of the projects
```

Comment:
let's add some words to differentiate from the old API ` /projects/{project_name}/logs`.
Maybe also updating the description of the old API to explicitly mention it's only for old data and will be deprecated?